README.md

```diff
@@ -41,110 +41,110 @@ Currently the following variables are supported:
 ### System Properties, `<system-properties></system-properties>`
 ```
 jbosseap_system_properties:
-- name: prop1
-  value: prop1val1
+  - name: prop1
+    value: prop1val1
 ```
 
 ### Logging, `<subsystem xmlns="urn:jboss:domain:logging">`
 ```
 jbosseap_logging:
-  logger:
-  - category: com.foo
-    level: DEBUG
-  formatter:
-  - name: myformatter
-    pattern: XXX
-  custom_handler:
-  - name: myhandler
-    class: com.bla
-    module: com.aa
-    formantter: XXX
-    level: XXX
-    properties: '("name1" => "val1"),("name2" => "val2")'
+    logger:
```

Comment:
All indents should be two spaces.  Example:

https://github.com/oasis-roles/firewalld#example-playbook